### PR TITLE
Handle empty leaderboard / reset leaderboard changes

### PR DESCRIPTION
**Pull request by KatLands**
Updated resetLeaderboard to use a raw query.

Added a check to weeklyLeaderboardResults that will look at the Users table and if empty, will not send the results to the server.

Added user specific tags to award reaction confirmation message.

### CODE REVIEW

File: tasks/tasks.js [hunk 1]
```diff
@@ -6,37 +8,44 @@ const { MessageEmbed } = require('discord.js'),
...
-Sec(0-59), min(0-59), hour(0-23), day of month(1-31), month(1-12), day of week(0-6 starting with sunday)
+Sec(0-59), min(0-59), hour(0-23), day of month(1-31), month(0-11), day of week(0-6 starting with sunday)
 */
 
+// 1 11 * * 1
 const weeklyLeaderboardResults = (client) => new CronJob('1 11 * * 1', async function() {
-    try {
-        const chartUrl = await getLeaderboardGraph();
+    const count = await sequelize.query('SELECT COUNT(*) FROM Users;');
+    if (count[0][0]['COUNT(*)'] > 0) {
```
Comment (aahmad4) [inline tasks/tasks.js:17]: Maybe in the future we can see if there's a cleaner way to extract the `count` from this query response

File: bot.js [hunk 2]
```diff
@@ -53,11 +53,11 @@ client.on('messageReactionAdd', async (reaction, user) => {
 
         if (foundUser) {
             foundUser.increment('points', { by: 5 });
-            reaction.message.channel.send(`Congrats ${foundUser.username}, you just earned 5 points! (Total points: ${foundUser.points + 5})`);
+            reaction.message.channel.send(`Congrats <@${reaction.message.author.id}>, <@${user.id}> just awarded you 5 points! (Total points: ${foundUser.points + 5})`);
```
Comment (aahmad4) [inline bot.js:56]: Do we want this and line 60 to be a shared variable maybe since they're both sending the same message?

File: tasks/tasks.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+const sequelize = require('../utilities/db');
```
Comment (aahmad4) [inline tasks/tasks.js:1]: nit: Can we combine this with the other `requires` right below?

File: tasks/tasks.js [hunk 4]
```diff
@@ -6,37 +8,44 @@ const { MessageEmbed } = require('discord.js'),
...
+// 1 11 * * 1
 const weeklyLeaderboardResults = (client) => new CronJob('1 11 * * 1', async function() {
-    try {
-        const chartUrl = await getLeaderboardGraph();
+    const count = await sequelize.query('SELECT COUNT(*) FROM Users;');
+    if (count[0][0]['COUNT(*)'] > 0) {
+        try {
+            const chartUrl = await getLeaderboardGraph();
 
-        const users = await Users.findAll({ limit: 5, order: [['points', 'DESC']] });
```
Comment (aahmad4) [inline tasks/tasks.js:16]: nit: Do we want to make this a raw SQL statement too just for consistency in the file?

File: tasks/tasks.js (hunk 3 above)
Comment (KatLands) [reply tasks/tasks.js]: Good idea. Made the change.

File: bot.js (hunk 2 above)
Comment (KatLands) [reply bot.js:56]: Sure, we can do that.

File: tasks/tasks.js (hunk 4 above)
Comment (KatLands) [reply tasks/tasks.js:16]: Sure, we can do that.

File: bot.js (hunk 2 above)
Comment (KatLands) [reply bot.js:56]: We could take the beginning portion of the message before the total points and set that to a shared variable. That would eliminate the redundancy.

**Pull request merged**